src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+GetDigitalCovid19CertificateTests.swift

```diff
@@ -0,0 +1,363 @@
+////
+// 🦠 Corona-Warn-App
+//
+
+@testable import ENA
+import Foundation
+import XCTest
+
+final class HTTPClientGetDigitalCovid19CertificateTests: CWATestCase {
+	
+	func testGIVEN_RegistrationToken_WHEN_HappyCase_THEN_DCCResponseIsReturned() throws {
+		// GIVEN
+		let registrationToken = "someToken"
+		
+		let expectedResponse = DCCResponse(
+			dek: "someKey",
+			dcc: "someCOSE"
```

Comment:
I find it irritating to see "dcc" assigned to something COSE-related. Until now I thought 'dcc' is something like 'digital covid certificate' and now _everything_ is dcc 🤯

Reply:
just implemented like the tech spec wanted 🤷‍♂️

Reply:
The Spec doesn't have to maintain this mess… 🙄